subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -504,6 +504,8 @@ include::sample[dir="snippets/artifacts/defineRepositoryInSettings/groovy",files
 include::sample[dir="snippets/artifacts/defineRepositoryInSettings/kotlin",files="settings.gradle.kts[tags=prefer_settings]"]
 ====
 
+[[sub:fail_build_on_project_repositories]]
```

Comment:
Why not add a subsection title as well?

Reply:
Because it breaks the reading flow. I wouldn't add a title every 2 lines..